tests/system/providers/microsoft/azure/example_azure_cosmosdb.py

```diff
@@ -48,12 +48,13 @@
     doc_md=__doc__,
     tags=["example"],
 ) as dag:
-
+    # [START cosmos_document_sensor]
     t1 = AzureCosmosDocumentSensor(
         task_id="check_cosmos_file",
         collection_name="airflow_example_coll",
         document_id="airflow_checkid",
     )
```

Comment:
The `database_name` parameter doesn't have a default value, you should add it.

Reply:
It is present at the DAG level - line 45

Reply:
Sorry, I didn't notice that.